Query Test flicker

> 
> https://github.com/statechannels/go-nitro/actions/runs/4133299374/attempts/1
> 
> ```
> -test.shuffle 1675938555609154846
> --- FAIL: TestQueryPaymentChannel (0.04s)
>     helpers_test.go:176: Payment channel diff mismatch (-want +got):
>           query.PaymentChannelInfo{
>           	ID:      {0xde, 0xbc, 0xad, 0x20, ...},
>         - 	Status:  "Closing",
>         + 	Status:  "Complete",
>           	Balance: {Payee: {0xbb, 0xb6, 0x76, 0xf9, ...}, Payer: {0xaa, 0xa6, 0x62, 0x8e, ...}, PaidSoFar: &{abs: {1}}, RemainingFunds: &{abs: {1}}},
>           }
> FAIL
> FAIL	github.com/statechannels/go-nitro/client_test	6.511s
> ```

--- a/client_test/rpc_test.go
+++ b/client_test/rpc_test.go
@@ -38,7 +38,7 @@ func TestRpcClient(t *testing.T) {
 
 	clientA, msgA := setupClientWithP2PMessageService(alice.PrivateKey, 3005, chainServiceA, createLogger(logDestination, "alice", ""))
 	clientB, msgB := setupClientWithP2PMessageService(bob.PrivateKey, 3006, chainServiceB, createLogger(logDestination, "bob", ""))
-	_, msgI := setupClientWithP2PMessageService(irene.PrivateKey, 3007, chainServiceI, createLogger(logDestination, "irene", ""))
+	clientI, msgI := setupClientWithP2PMessageService(irene.PrivateKey, 3007, chainServiceI, createLogger(logDestination, "irene", ""))
 	peers := []p2pms.PeerInfo{
 		{Id: msgA.Id(), IpAddress: "127.0.0.1", Port: 3005, Address: alice.Address()},
 		{Id: msgB.Id(), IpAddress: "127.0.0.1", Port: 3006, Address: bob.Address()},
@@ -63,10 +63,12 @@ func TestRpcClient(t *testing.T) {
 	// Quick sanity check that we're getting a valid objective id
 	assert.Regexp(t, "DirectFunding.0x.*", res.Id)
 	waitTimeForCompletedObjectiveIds(t, &clientA, defaultTimeout, res.Id)
+	waitTimeForCompletedObjectiveIds(t, &clientI, defaultTimeout, res.Id)
 
 	bobResponse := clientB.CreateLedgerChannel(irene.Address(), 100, testdata.Outcomes.Create(bob.Address(), irene.Address(), 100, 100, types.Address{}))
 
 	waitTimeForCompletedObjectiveIds(t, &clientB, defaultTimeout, bobResponse.Id)
+	waitTimeForCompletedObjectiveIds(t, &clientI, defaultTimeout, bobResponse.Id)
 
 	vRes := rpcClientA.CreateVirtual(
 		[]types.Address{irene.Address()},
@@ -76,13 +78,17 @@ func TestRpcClient(t *testing.T) {
 
 	assert.Regexp(t, "VirtualFund.0x.*", vRes.Id)
 	waitTimeForCompletedObjectiveIds(t, &clientA, defaultTimeout, vRes.Id)
-
+	waitTimeForCompletedObjectiveIds(t, &clientB, defaultTimeout, vRes.Id)
+	waitTimeForCompletedObjectiveIds(t, &clientI, defaultTimeout, vRes.Id)
 	rpcClientA.Pay(vRes.ChannelId, 1)
 
 	closeVId := rpcClientA.CloseVirtual(vRes.ChannelId)
 	waitTimeForCompletedObjectiveIds(t, &clientA, defaultTimeout, closeVId)
+	waitTimeForCompletedObjectiveIds(t, &clientB, defaultTimeout, closeVId)
+	waitTimeForCompletedObjectiveIds(t, &clientI, defaultTimeout, closeVId)
 
 	closeId := rpcClientA.CloseLedger(res.ChannelId)
 	waitTimeForCompletedObjectiveIds(t, &clientA, defaultTimeout, closeId)
+	waitTimeForCompletedObjectiveIds(t, &clientI, defaultTimeout, closeId)
 
 }

